backend/server/app.py

```diff
@@ -8,7 +8,7 @@
 import os
 
 
-app = Flask(__name__)
+app = Flask(__name__, static_url_path='/api/')
```

Comment:
having static files in the /api/ URL is a bit unexpected for me. I guess we may need to clean this up in the future. in general it's ok for now